ezyhttp-core/src/main/java/com/tvd12/ezyhttp/core/response/ResponseEntity.java

```diff
@@ -13,130 +13,110 @@
 import lombok.Getter;
 
 @Getter
-public class ResponseEntity {
+public class ResponseEntity<T> {
```

Comment:
I can guess you've using spring, and you want to make it like spring, but, let's thinking about this case:

```
@DoPost("/add")
public ResponseEntity addCustomer(@RequestBody Customer customer) {
	if(customer.getName() == null)
		return ResponseEntity.badRequest();
	customerService.save(customer);
	return ResponseEntity.builder().body(customer).build();
}
```

You can see, there are many type of response body. Actually, you just use `ResponseEntity `is enough